godel_process_planning/src/blend_process_planning.cpp

```diff
@@ -0,0 +1,157 @@
+#include <godel_process_planning/godel_process_planning.h>
+
+#include <ros/console.h>
+
+// descartes
+#include "descartes_trajectory/axial_symmetric_pt.h"
+#include "descartes_trajectory/joint_trajectory_pt.h"
+#include "descartes_planner/dense_planner.h"
+
+#include "common_utils.h"
+
+
+namespace godel_process_planning
+{
+
+/**
+ * @brief toDescartesPt
+ * @param pose
+ * @return
+ */
+static inline descartes_core::TrajectoryPtPtr toDescartesPt(const Eigen::Affine3d& pose, double dt)
+{
+  using namespace descartes_trajectory;
+  using namespace descartes_core;
+  const descartes_core::TimingConstraint tm (dt);
+  return TrajectoryPtPtr(new AxialSymmetricPt(pose, M_PI/12.0, AxialSymmetricPt::Z_AXIS, tm)); // default timing
```

Comment:
Couple of things:
1. Should use `boost::make_shared` as it is both simpler to read and more efficient (1 allocation for a shared pointer versus 2 for this method).
2. The `M_PI/12.0` is a magic number. Should at least be a constant, but probably should be an argument to this function.